### ISSUE TITLE
Disable Firefox 63 Update Check [question]

### ISSUE THREAD

**Issue by ldgbc** [cadeyrn/enterprise-policy-generator#44]
Hi, have to learn this new Group Policy thing due to Mozilla decision. I'm currently using Firefox 63 where Update Check is automatic. I understand that you can disable that using Group Policy.

This add-on allow me to do that it seem, but not sure which option it is, some of the option sound like it disable update all together. I just want to disable the Update Check and manually check and update when I want. Can I do that with add-on yet?

Basically this one:

> Never check for updates option removed from about:preferences. If a substitute is needed, we recommend using the DisableAppUpdate enterprise policy
https://www.mozilla.org/en-US/firefox/63.0beta/releasenotes/

**Comment by ldgbc** [comment]
Nevermind it is this option here: 

"Prevent Firefox from updating", but that tooltips is misleading.
